.github/workflows/test.yml

```diff
@@ -1,23 +1,25 @@
 name: Test & Lint
 
-on:
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
+on: [push, pull_request]
```

Comment:
Is there a reason to run this pipeline only on main branch? (otherwise we don't have the build results in PRs)